trackbasket_be/models/at_risk_user.py

```diff
@@ -0,0 +1,24 @@
+from models.basemodel import BaseModel, db
+import datetime 
+
+class AtRiskUser(BaseModel, db.Model):
+  __tablename__ = 'at_risk_users'
+  id = db.Column(db.Integer, primary_key=True)
+  name = db.Column(db.String)
+  at_risk_user_id = db.Column(db.String)
+  address = db.Column(db.String)
+  city = db.Column(db.String)
+  state = db.Column(db.String)
+  zipcode = db.Column(db.String)
+  phone_number = db.Column(db.String)
+
+  def json(self):
+    return {'data': { 'id': 'at_risk_user', 'attributes': {'name': self.name, 'at_risk_user_id': self.at_risk_user_id, 'address': self.address, 'city': self.city, 'state': self.state, 'zip code': self.zipcode, 'phone number': self.phone_number} } }
+
+  @classmethod 
+  def find_by_id(cls, id):
+    return cls.query.filter_by(at_risk_user_id=id).first()                                                                                                                                            
```

Comment:
I like this helper method, hopefully we have time to refactor and turn that into a module and use it for other models as well